common/client/entrypoints/renderer/src/pages/_app.tsx

```diff
@@ -1,3 +1,5 @@
+import 'cropperjs/dist/cropper.css'
```

Comment:
Такая ошибка появляется:
```
../../../../../.yarn/cache/cropperjs-npm-1.6.1-cfaf521c79-4865c5c883.zip/node_modules/cropperjs/dist/cropper.css
Global CSS cannot be imported from files other than your Custom <App>. Due to the Global nature of stylesheets, and to avoid conflicts, Please move all first-party global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://nextjs.org/docs/messages/css-global
```

Reply:
Так ты не импортируй как файл целиком, а перенеси в styles.ts как мы делали в Ганте.